docs/content/latest/quick-start/build-apps/java/ysql-ebeans.md

```diff
@@ -0,0 +1,369 @@
+      YSQL - Ebeans
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  <li>
+    <a href="../ycql-4.6/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL (4.6)
+    </a>
+  </li>
+</ul>
+
+The following tutorial implements an ORM example using [Ebeans](https://ebean.io/docs/), an ORM in Java, that implements a simple REST API server. The scenario is that of an e-commerce application where the database access is managed through [Play framework](https://www.playframework.com/documentation/2.8.x/api/java/index.html); Play uses [Akka](https://doc.akka.io/docs/akka/current/typed/guide/introduction.html) internally and exposes Akka Streams and actors in Websockets and other streaming HTTP responses. It includes the following tables:
+
+- `users` table — the users of the e-commerce site
+- `products` table — the products being sold
+- `orders` table — the orders placed by the users
+- `orderline` table — each line item of an order
+
+The source for the above application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master/java/ebeans) repository.
+
+## Prerequisites
+
+This tutorial assumes that you have:
+
+- YugabyteDB up and running. If you are new to YugabyteDB, follow the steps in [Quick start](../../../../quick-start/) to have YugabyteDB up and running in minutes.
+- Java Development Kit (JDK) 1.8 is installed. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/).
+- [sbt](https://www.scala-sbt.org/1.x/docs/) is installed.
+
+## Clone the "orm-examples" repository
+
+```sh
+$ git clone https://github.com/yugabyte/orm-examples.git
+```
+
+## Database configuration
+
+- Modify the database configuration section to include YugabyteDB's JDBC driver settings in `conf/application.conf` file as follows:
+
+  ```sh
+  #Default database configuration using PostgreSQL database engine
+  default.username=yugabyte
+  default.password=""
+  default.driver=com.yugabyte.Driver
+  default.url="jdbc:yugabytedb://127.0.0.1:5433/ysql_ebeans?load-balance=true"
+  ```
+
+- Add a dependency in `build.sbt` for YugabyteDB’s JDBC driver.
+
+  ```sh
+  libraryDependencies += "com.yugabyte" % "jdbc-yugabytedb" % "42.3.0"
+  ```
+
+- From your local YugabyteDB installation directory, connect to the [YSQL] shell to create a database.
+
+  ```sh
+  bin/ysqlsh
+  CREATE DATABASE ysql_ebeans;
+  ```
+
+## Build the application
+
+Build the REST API server using:
+
+```sh
+$ sbt compile
+```
+
+{{< note title="Note" >}}
+
+- Certain subversions of JDK 1.8 would require the `nashorn` package. Add the package in the `build.sbt` if there is a compile error regarding a non-existing `jdk.nashorn` package.
```

Comment:
The reason I added it here was because adding the package is only a requirement if there is a compile error. I believe @priyanshi-yb did not even see these errors at her end. So I think , adding this note as well as the entry for 'sbt.version = 1.2.8' in the build.properties file should remain here.

Reply:
Yah that's true and I too think this can be here only because this is required only when someone faces this issue.